pkg/rancher-desktop/store/diagnostics.ts

```diff
@@ -57,7 +57,7 @@ const mapMarkdownToDiagnostics = async(diagnostics: DiagnosticsResult[]) => {
  * from the provided markdown.
  */
 const markdown = async(raw: string) => {
-  const markedString = await marked.parseInline(raw);
+  const markedString = await (marked.parseInline(raw) as Promise<string>);
```

Comment:
This is in the wrong PR, and also completely unnecessary.  You probably need to `yarn install` to drop the excess `@types/marked`.